src/styles/form-controls/_checkbox.scss

```diff
@@ -7,6 +7,8 @@ div.alpheios-checkbox-block {
 
   max-width: uisize(235px);
   vertical-align: top;
+
+  min-height: calc(var(--alpheios-base-ui-size) * 1.375);
```

Comment:
maybe also requires #439 ?

Reply:
It is for the problem with invisible check on the panel - I will check before merging.

Reply:
No it is not, so I won't remove this check.